File: ServerET/ServerET/Models/Student.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace ServerET.Models
{
    public class Student
    {
        public string Id { get; set; }  
        public string LastName { get; set; }
        public string FirstName { get; set; }
        public string Otchestvo { get; set; }
        public int IsLogIn { get; set; } //1 - в системе , 0 - вышел из системы 

        public int GroupId { get; set; }
        public Group Group { get; set; }

        public List<Result> Results { get; set; } = new List<Result>();
    }
}

File: login_app/app/src/main/java/com/example/login_app/api/service/Student.kt
package com.example.login_app.api.service

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Student {

    @SerializedName("Id")
    @Expose
    private var id : String? = null

    @SerializedName("LastName")
    @Expose
    private var lastName: String? = null

    @SerializedName("FirstName")
    @Expose
    private var name: String? = null

    @SerializedName("Otchestvo")
    @Expose
    private var otch: String? = null

    @SerializedName("IsLogIn")
    @Expose
    private var isLogIn  = 0

    @SerializedName("GroupId")
    @Expose
    private var groupId  = 0

    fun getId(): String? {
        return id
    }

    fun setId(Id: String) {
        this.id = id
    }

    fun getLastName(): String? {
        return lastName
    }

    fun setLastName(lastName: String?) {
        this.lastName = lastName
    }

    fun getName(): String? {
        return name
    }

    fun setName(body: String?) {
        this.name = body
    }

    fun getOtchestvo(): String? {
        return otch
    }

    fun setOtchestvo(otch: String?) {
        this.otch = otch
    }

    fun getIsLogIn(): Int {
        return isLogIn
    }

    fun getGrouId(): Int {
        return groupId
    }

    fun setGroupId(groupId: Int) {
        this.groupId = groupId
    }

}


